src/utils/isWeekend.ts

```diff
@@ -0,0 +1,3 @@
+export default function isWeekend(targetDate: Date) {
```

Comment:
why don't use `isWeekend` in date-fns?

https://date-fns.org/v2.21.3/docs/isWeekend

Reply:
Reflected it.

Reply:
we don't need to this function! :)